py-phylo2vec/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 use pyo3::prelude::*;
 
 pub use::phylo2vec::to_newick as _to_newick;
```

Comment:
```suggestion
pub use phylo2vec::to_newick as _to_newick;
```
Looks like the non change code has an incorrect syntax here 😅 definitely not your fault haha